.github/workflows/release-version.yml

```diff
@@ -106,11 +106,18 @@ jobs:
         with:
           github-token: ${{ steps.get-token.outputs.token }}
           script: |
-            const { releaseVersionComment } = require('./ci/pull-request-comments')
+            const body = `Once merged, this PR will automatically create a GitHub release for you.
+            The description of the release will exactly match this PR's description. Feel free to edit it.
+
+            > [!WARNING]
+            > The release will be created **but not published** on the GitHub Marketplace (because the API doesn't allow that).
+            >
+            > To publish it manually, please edit the release [here](https://github.com/DataDog/synthetics-ci-github-action/releases/edit/${{ steps.bump-version.outputs.NEW_VERSION_TAG }}).
+            > The \`Publish this Action to the GitHub Marketplace\` checkbox will be checked by default.`
```

Comment:
You mention `edit the release`, but you don't mention what they should do after they edit it, or any extra information about editing it. Could you provide a very brief statement, like "After setting the version field to the target release version, click `Publish` to release the new version to the official platform." Something that customers will know is the final step.

Reply:
Actually there is _nothing_ to do, you just have to go on the edit page, and save. The checkbox that you have to check is checked by default in the form.

I edited the text to "Make sure [it's] checked, then save".